CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+- [#450](https://github.com/embedded-graphics/embedded-graphics/pull/450) Added `RawTga` to use `tinytga` without using a embedded-graphic color type.
+- [#450](https://github.com/embedded-graphics/embedded-graphics/pull/450) Added `Tga::from_raw` to convert a `RawTga` into a `Tga` object.
+- [#450](https://github.com/embedded-graphics/embedded-graphics/pull/450) Added `DynamicTga` to allow drawing of TGA images without a known color format at compile time.
+
+### Fixed
+
+- [#407](https://github.com/embedded-graphics/embedded-graphics/pull/407) Additional data in `pixel_data`, beyond `width * height` pixels, is now discarded by `TgaIterator`.
+- [#430](https://github.com/embedded-graphics/embedded-graphics/pull/430) Images with unsupported BPP values in the header no longer cause panics. Instead an error is returned by `Tga::from_slice`.
+- [#430](https://github.com/embedded-graphics/embedded-graphics/pull/430) Errors during the execution of a pixel iterator no longer cause panics. Instead the corrupted portion of the image is filled with black pixels.
+
+## [0.3.2] - 2020-03-20
+
+## [0.3.1] - 2020-02-17
+
+- **(breaking)** [#247](https://github.com/embedded-graphics/embedded-graphics/pull/247) "reverse" integration of tinytga into [`embedded-graphics`](https://crates.io/crates/embedded-graphics). tinytga now has a `graphics` feature that must be turned on to enable embedded-graphics support. The `tga` feature from embedded-graphics is removed.
+
+  **Before**
+
+  `Cargo.toml`
+
+  ```toml
+  [dependencies]
+  embedded-graphics = { version = "0.6.0-alpha.3", features = [ "tga" ]}
+  ```
+
+  Your code
+
+  ```rust
+  use embedded_graphics::prelude::*;
+  use embedded_graphics::image::ImageTga;
+
+  let image = ImageTga::new(include_bytes!("../../../assets/patch.tga")).unwrap();
+  display.draw(&image);
+  ```
+
+  **After**
+
+  `Cargo.toml`
+
+  ```toml
+  [dependencies]
+  embedded-graphics = "0.6.0"
+  tinytga = { version = "*", features = [ "graphics" ]}
+  ```
+
+  Your code
+
+  ```rust
+  use embedded_graphics::{prelude::*, image::Image};
+  use tinytga::Tga;
+
+  let image = Tga::new(include_bytes!("../../../assets/patch.tga")).unwrap();
+  let image = Image::new(&image);
+  display.draw(&image);
+  ```
+
+## 0.2.0
+
+### Added
+
+- [#217](https://github.com/embedded-graphics/embedded-graphics/pull/217) Added support for TGA files with color map.
+
+### Fixed
+
+- [#217](https://github.com/embedded-graphics/embedded-graphics/pull/217) Images without a TGA footer are now parsed correctly.
+- [#216](https://github.com/embedded-graphics/embedded-graphics/pull/216) Fixed integer overflow for some RLE compressed TGA files.
+- [#218](https://github.com/embedded-graphics/embedded-graphics/pull/218) Test README examples in CI and update them to work with latest crate versions.
+
+<!-- next-url -->
+
+[unreleased]: https://github.com/embedded-graphics/tinytga/compare/v0.3.2...HEAD
```

Comment:
Sorry to leave yet another comment for this section, but I hadn't noticed this before: `v0.3.2` isn't the correct name for the tag IMO, because the content of the repo is newer than that version. Maybe we could add an `after-split` tag to this repo?

Reply:
No worries, I've pushed a new `after-split` tag on tinybmp, tinytga and simulator.